cairo/tests/fixtures/runner.py

```diff
@@ -272,3 +302,9 @@ def _factory(entrypoint, *args, **kwargs):
         return final_output
 
     return _factory
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
Why not in conf test and no `@pytest.hookimpl()` ?
How is it discovered and called ?
It is not well documented in pytest it seems